src/Tizen.Multimedia.Vision/MediaVision/MediaVisionSource.cs

```diff
@@ -217,6 +217,26 @@ public ColorSpace Colorspace
             }
         }
 
+        /// <summary>
+        /// Gets or sets the timestamp of the media source.
+        /// </summary>
+        /// <exception cref="ObjectDisposedException">The <see cref="MediaVisionSource"/> has already been disposed.</exception>
+        /// <since_tizen> 11 </since_tizen>
+        public ulong Timestamp
```

Comment:
please consider changing `ulong` to .Net type `UInt64`, those are aliases of the same type but the UInt64 is more explicit which is better for public API

Reply:
It's same type so I just want to keep alias type - uint.